test/cli/template-test/local/expected-stderr

```diff
@@ -0,0 +1 @@
+Error: not implemented
```

Comment:
vrátí chybu z https://github.com/keboola/keboola-as-code/blob/9ae1aada967083cf445fc435a74f8b508ad1b032/pkg/lib/operation/template/local/test/operation.go#L19

Reply:
jinak tu není nic zajímavýho. 😄 Zbytek souborů je jen k tomu aby to běželo. V `in` i `out` je jen jedna šablona nad kterou se později budou spouštět ty testy. (je to copy&paste z `test/cli/template-describe/describe` testu)